<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDeviceDialog</class>
 <widget class="QDialog" name="OpenDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Открыть устройство</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceDescriptionText">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>13</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите устройство</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>310</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>307</y>
     <width>95</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>307</y>
     <width>95</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="deviceNameLineEdit">
      <property name="placeholderText">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="devicesListView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>deviceNameLineEdit</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>devicesListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
